TypeScript/src/components/Part-B-Functions.ts

```diff
@@ -0,0 +1,81 @@
+import { Order }  from './Order';
+import {  OrderInterface, orderInformation } from './Part-A';
+import { billOutput } from '../output/Output';
+let array : OrderInterface[] = [];
+type MaybeOrderInterface = OrderInterface | undefined;
+
+const orderObjects = () : OrderInterface[]=>{
+    return array;
+}
+   
+const addOrder = (number : number) : void =>{
+    const order1 : Order = new Order(number);
+    const orderObject : OrderInterface = orderInformation(order1);
+    array = [...array,orderObject];
+}
+
+const printOrders = () : void  =>{
+    if (array.length === 0){
+        console.log('No Orders yet!!')
+    }
+    else{
+        console.log(array);
+    }
+}
+
+
+const deleteOrder = (number : number) : void =>{
```

Comment:
let's try not naming variables like types

Reply:
ok sure